js/bkex.js

```diff
@@ -232,52 +254,90 @@ module.exports = class bkex extends Exchange {
+        //                 "minimumTradeVolume": "0E-18",
+        //                 "pricePrecision": "11",
+        //                 "supportTrade": true,
+        //                 "symbol": "COMT_USDT",
+        //                 "volumePrecision": 0
+        //             },
+        //         ],
+        //         "msg": "success",
+        //         "status": 0
+        //     }
         //
-        const data = this.safeValue (response, 'data', []);
+        const swapMarkets = resolved[1];
+        //
+        //     {
+        //         "code": 0,
+        //         "msg": "success",
+        //         "data": [
+        //             {
+        //                 "symbol": "luna_usdt",
+        //                 "supportTrade": false,
+        //                 "volumePrecision": 0,
+        //                 "pricePrecision": 3,
+        //                 "marketMiniAmount": "1",
+        //                 "limitMiniAmount": "1"
+        //             },
+        //         ]
+        //     }
+        //
+        const spotData = this.safeValue (spotMarkets, 'data', []);
+        const swapData = this.safeValue (swapMarkets, 'data', []);
+        const data = this.arrayConcat (spotData, swapData);
         const result = [];
         for (let i = 0; i < data.length; i++) {
             const market = data[i];
-            const id = this.safeString (market, 'symbol');
+            const marketId = this.safeString (market, 'symbol');
+            const id = this.safeStringUpper (market, 'symbol');
             const [ baseId, quoteId ] = id.split ('_');
             const base = this.safeCurrencyCode (baseId);
             const quote = this.safeCurrencyCode (quoteId);
+            const minimumOrderSize = this.safeString (market, 'minimumOrderSize');
+            const type = (minimumOrderSize !== undefined) ? 'spot' : 'swap';
+            const swap = (type === 'swap');
+            let symbol = base + '/' + quote;
+            let settleId = undefined;
+            let settle = undefined;
+            if (swap) {
+                settleId = quoteId;
+                settle = quote;
+                symbol = base + '/' + quote + ':' + settle;
+            }
             result.push ({
-                'id': id,
-                'symbol': base + '/' + quote,
+                'id': marketId,
+                'symbol': symbol,
                 'base': base,
                 'quote': quote,
-                'settle': undefined,
+                'settle': settle,
                 'baseId': baseId,
                 'quoteId': quoteId,
-                'settleId': undefined,
-                'type': 'spot',
-                'spot': true,
+                'settleId': settleId,
+                'type': type,
+                'spot': (type === 'spot') ? true : false,
                 'margin': false,
                 'future': false,
-                'swap': false,
+                'swap': swap ? true : false,
```

Comment:
same here, if `swap` is already a bool we don't need to do `if true return true else return false`